libminifi/include/io/StreamPipe.h

```diff
@@ -42,6 +42,11 @@ class OutputStreamCallback {
   virtual ~OutputStreamCallback() = default;
   virtual int64_t process(const std::shared_ptr<io::BaseStream>& stream) = 0;
 };
+class InputOutputStreamCallback {
+ public:
+  virtual ~InputOutputStreamCallback() = default;
+  virtual int64_t process(const std::shared_ptr<io::BaseStream>& input, const std::shared_ptr<io::BaseStream>& output) = 0;
+};
```

Comment:
Is this what you mean: https://github.com/fgerlits/nifi-minifi-cpp/commit/cdfbca12e6e2a49f93f286f411cecd91a0bce1bf?  I will do it if you explain why it would be better.

Reply:
As discussed, let's keep it as it is for now, and possibly change all three (read, write, readWrite) functions together in a separate PR.